week-four/app/src/test/java/pro/marvinhosea/movielist/ui/movies/show/MovieDetailViewModelTest.kt

```diff
@@ -0,0 +1,33 @@
+package pro.marvinhosea.movielist.ui.movies.show
+
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.MainScope
+import kotlinx.coroutines.launch
+import org.junit.Assert.*
+import org.junit.Test
+import pro.marvinhosea.movielist.data.models.Movie
+
+class MovieDetailViewModelTest {
+    private val movieDetailViewModel by lazy { MovieDetailViewModel() }
+    var movie = Movie(
+        3452,
+        "Test Movie",
+        "This is a test movie",
+        7.0,
+        null,
+        "Action",
+        "3.04.2020",
+        false,
+        "marvin"
+    )
+
+    @Test
+    fun testCanAddMovieWithNoPosterLinkToWatchlist() {
```

Comment:
Again, this test doesn't really test anything. I mean, it does in the sense that it does what you say, but no mocks, no verify, coroutines are flaky.